docker/bitcoin.conf

```diff
@@ -0,0 +1,45 @@
+[signet]
+# Signet network configuration for running Coinswap Taker and Maker
+# Mutinynet default signet parameters
+signetchallenge=512102f7561d208dd9ae99bf497273e16f389bdbd6c4742ddb8e6b216e64fa2928ad8f51ae
+addnode=45.79.52.207:38333
```

Comment:
Lets keep this in the doc folder for better visibility. It's not only for docker. Needs to be used for running manual setup also on mutinynet and is linked from the `bitcoind.md`.